### ISSUE TITLE
🔗 🐞 Bug Report: Header cannot be edited after adding mega menu (Duplicate of #21956) - [ED-10424]

### ISSUE THREAD

**Issue by otomasz** [elementor/elementor#22108]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The problem is that the header cannot be edited. The header cannot be edited even in safe mode. I have tried all possible ways.

### Steps to reproduce

1. Create a headline
2. Add a mega menu
3. Exit the editor
4. Try editing the header again

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.5.11
	PHP Version: 8.1.16
	PHP Memory Limit: 8192M
	PHP Max Input Vars: 20000
	PHP Max Post Size: 8192M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://prostyformat.pl
	Home URL: https://prostyformat.pl
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 40M
	Max Memory limit: 8192M
	Permalink Structure: /%postname%/
	Language: pl-PL
	Timezone: Europe/Warsaw
	Admin Email: tomasz@pomelomedia.pl
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: pl_PL
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36

== Active Plugins ==
	Activity Log
		Version: 2.8.5
		Author: Activity Log Team

	Autoptimize
		Version: 3.1.6
		Author: Frank Goossens (futtta)

	Converter for Media
		Version: 5.8.5
		Author: matt plugins

	Easy HTTPS (SSL) Redirection
		Version: 1.9.1
		Author: Tips and Tricks HQ

	Elementor
		Version: 3.12.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.2
		Author: Elementor.com

	Icon Element
		Version: 2.0.1
		Author: Webangon

	JetEngine
		Version: 3.1.4
		Author: Crocoblock

	JetFormBuilder
		Version: 3.0.3
		Author: Crocoblock

	JetProductGallery
		Version: 2.1.11
		Author: Crocoblock

	JetSmartFilters
		Version: 3.0.4
		Author: Crocoblock

	ManageWP - Worker
		Version: 4.9.17
		Author: GoDaddy

	Orphans
		Version: 3.1.2
		Author: Marcin Pietrzak

	Post to Google My Business
		Version: 3.1.11
		Author: Koen Reus

	Post Types Order
		Version: 2.0.5
		Author: Nsp Code

	Rank Math SEO
		Version: 1.0.111
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.33
		Author: Rank Math

	White Label CMS
		Version: 2.5
		Author: www.videousermanuals.com

	WPCode Lite
		Version: 2.0.9
		Author: WPCode

	WP Fastest Cache
		Version: 1.1.4
		Author: Emre Vona

	WPS Hide Login
		Version: 1.9.8
		Author: WPServeur, NicolasKulka, wpformation

	WPS Limit Login
		Version: 1.5.6
		Author: WPServeur, NicolasKulka, wpformation

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

	ManageWP - Worker Loader
		Version: 
		Author: GoDaddy


== Wykorzystanie elementów ==
	
	archive : 2
		container : 11
		heading : 3
		html : 1
		icon : 1
		jet-listing-grid : 1
		jet-smart-filters-radio : 1
		spacer : 17
	container : 5
		button : 8
		container : 87
		divider : 2
		heading : 74
		html : 1
		icon : 12
		icon-list : 3
		image : 14
		jet-listing-dynamic-terms : 1
		jet-listing-grid : 2
		jet-smart-filters-radio : 1
		jet-woo-product-gallery-grid : 1
		spacer : 43
		star-rating : 8
		text-editor : 2
	footer : 1
		button : 1
		container : 15
		heading : 8
		icon-list : 2
		image : 2
		spacer : 5
	page : 2
		heading : 6
		icon-list : 1
		spacer : 5
	wp-post : 0
		container : 5
		heading : 5
	wp-page : 16
		button : 30
		container : 410
		form : 2
		heading : 305
		icon : 25
		icon-box : 2
		icon-list : 17
		image : 62
		jet-form-builder-form : 1
		jet-listing-grid : 8
		nested-tabs : 5
		spacer : 179
		star-rating : 5
		template : 5
		text-editor : 73
	single-post : 2
		container : 14
		divider : 1
		form : 1
		heading : 11
		icon : 5
		jet-listing-dynamic-terms : 1
		jet-woo-product-gallery-grid : 1
		spacer : 3
		theme-post-content : 1
	jet-listing-items : 0
		container : 1
		heading : 1
		image : 1
		jet-listing-dynamic-terms : 1


== Ustawienia ==
	
	cpt_support: page
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: block
	meta_generator_tag: 1


== Składniki oferty ==
	Custom Fonts: 1
	Custom Icons: 0

== Integracje ==
	


== Eksperymenty z Elementorem ==
	Zoptymalizowany wynik DOM: Domyślnie aktywne
	Ulepszone ładowanie zasobów: Włączono
	Poprawione ładowanie CSS: Włączono
	Czcionka ikon: Domyślnie nieaktywne
	Ulepszenia w zakresie dostępności: Domyślnie aktywne
	Dodatkowe niestandardowe punkty zmiany: Włączono
	admin_menu_rearrangement: Domyślnie nieaktywne
	Flexbox Container: Włączono
	Upgrade Swiper Library: Włączono
	Default to New Theme Builder: Domyślnie aktywne
	Hello Theme Header & Footer: Domyślnie nieaktywne
	Editor Top Bar: Nieaktywne
	Strony docelowe: Włączono
	Nested Elements: Włączono
	Lazy Load Background Images: Domyślnie nieaktywne
	Page Transitions: Nieaktywne
	Notes: Domyślnie aktywne
	Loop: Domyślnie aktywne
	Form Submissions: Domyślnie aktywne
	Scroll Snap: Nieaktywne
	Menu: Włączono


== Dziennik ==
	
Log: showing 20 of 262023-03-31 17:07:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.0',
)]
2023-03-31 17:10:38 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.0',
)]
2023-03-31 17:10:40 [info] elementor-pro::elementor_pro_updater Started 
2023-03-31 17:10:40 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-31 17:10:40 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-31 17:10:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.0',
)]
2023-04-02 19:06:44 [info] elementor::elementor_updater Started 
2023-04-02 19:06:44 [info] Elementor/Upgrades - _on_each_version Start  
2023-04-02 19:06:45 [info] Elementor/Upgrades - _on_each_version Finished 
2023-04-02 19:06:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.0',
  'to' => '3.12.1',
)]
2023-04-02 20:07:33 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.0',
  'to' => '3.12.1',
)]
2023-04-02 20:07:34 [info] elementor-pro::elementor_pro_updater Started 
2023-04-02 20:07:34 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-02 20:07:34 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-02 20:07:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.0',
  'to' => '3.12.1',
)]
2023-04-10 20:26:03 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-04-10 20:26:03 [info] elementor-pro::elementor_pro_updater Started 
2023-04-10 20:26:03 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-10 20:26:03 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-10 20:26:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]

JS: showing 10 of 10JS: 2023-04-01 14:00:58 [error X 1][https://prostyformat.pl/wp-content/plugins/elementor/assets/js/df16c9d56464dd4f0800.bundle.min.js:2:7826] Cannot read properties of null (reading 'dataset') 
JS: 2023-04-02 09:01:44 [error X 4][https://prostyformat.pl/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.12.0:2:54646] Model.isValidChild() should be implemented, please provide 'isValidChild' functionality. 
JS: 2023-04-02 11:19:15 [error X 38][https://prostyformat.pl/wp-admin/post.php?post=1736&action=elementor#:68:18] Assignment to constant variable. 
JS: 2023-04-02 11:29:24 [error X 1][https://prostyformat.pl/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view4891") has already been destroyed and cannot be used. 
JS: 2023-04-02 12:14:04 [error X 1][https://prostyformat.pl/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view23020") has already been destroyed and cannot be used. 
JS: 2023-04-02 14:07:01 [error X 1][https://prostyformat.pl/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view16179") has already been destroyed and cannot be used. 
JS: 2023-04-03 16:06:18 [error X 1][https://prostyformat.pl/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.1:3:594206] Cannot read properties of null (reading 'id') 
JS: 2023-04-03 16:27:30 [error X 3][https://prostyformat.pl/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.1:3:673293] this.model.isValidChild is not a function 
JS: 2023-04-03 18:31:24 [error X 1][https://prostyformat.pl/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view4989") has already been destroyed and cannot be used. 
JS: 2023-04-05 16:50:56 [error X 4][undefined:67:24] Attempted to assign to readonly property. 

PHP: showing 6 of 6PHP: 2023-04-02 19:09:06 [warning X 5][/home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-02 19:09:08 [notice X 8262][/home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::204] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-05 19:03:28 [notice X 2][/home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-content/plugins/elementor/includes/template-library/sources/local.php::747] ob_end_clean(): Failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-10 20:26:23 [warning X 17][/home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-content/plugins/elementor/core/files/file-types/svg.php::191] SimpleXMLElement::__construct(): 	                      ^ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-10 20:27:53 [notice X 11][/home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-16 09:23:34 [notice X 1][/home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-content/plugins/elementor-pro/modules/custom-css/module.php(76): class type trim()
#2: /home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-includes/class-wp-hook.php(308): ElementorPro\Modules\CustomCss\Module -> add_page_settings_css()
#3: /home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-includes/class-wp-hook.php(332): WP_Hook -> apply_filters()
#4: /home/klient.dhosting.pl/oomaster/prostyformat.pl/public_html/wp-includes/plugin.php(517): WP_Hook -> do_action()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Niekompatybilne
	JetProductGallery: Niekompatybilne

== Elementor Pro - Compatibility Tag ==
	
	JetProductGallery: Niekompatybilne
				
````

</details>

**Comment by Fourmarketing360** [comment]
Happening here too

**Comment by emunot** [comment]
I am having same exact issue and it's been driving me crazy thinking I did something wrong. Will there be any fix anytime soon?

**Comment by emunot** [comment]
This issue seems to have been resolved in the recent update. I can now edit my header.

**Comment by Fourmarketing360** [comment]
Same here, it seems its working right now using elementor 3.12.2 and Pro 3.12.3

**Comment by leonyl-ber** [comment]
Hello Git Hubber,

what a shock this morning, I have the same error on one of my websites.

I could not create a header and I am working with Hello Elementor theme. The instruction: 1. Create a headline - did not work, even though I disabled all other plugins.
**My solution:** I had already defined a header in the Theme Customizer under Website Settings and turned the menu off and on again there.
Now it works

Hope it's work also for you.

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #21956

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Best regards,

Nick

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved with the release of Elementor v3.23.0! 🥳 

The time to interaction should improve by 60%-98%, and performance overall should be better. For a full breakdown please read [this article](https://developers.elementor.com/elementor-3-23-developers-update/#:~:text=Performant%20Nested%20Elements%20In%20the%20Editor).

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
